### DISCUSSION TITLE
初识Unity Input System - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#11] [Announcements]
# 初识Unity Input System - 没有秃头基因的燊的博客

在一次想做鼠标拖拽摄像机旋转的时候，了解到Unity Input System，就大致介绍一下Unity Input System的基本使用以及如何实现一个简单的右键拖拽控制相机旋转 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/54306/

**Comment by Furry-Monster** [comment]
感谢，很有帮助喵~ (●'◡'●)
